fmriprep/workflows/confounds.py

```diff
@@ -7,36 +7,46 @@
 Calculates frame displacement, segment regressors, global regressor, dvars, aCompCor, tCompCor
 '''
 from __future__ import print_function, division, absolute_import, unicode_literals
-
 from niworkflows.nipype.interfaces import utility
+from niworkflows.nipype.interfaces import fsl
 from niworkflows.nipype.algorithms import confounds
 from niworkflows.nipype.pipeline import engine as pe
+from niworkflows.nipype.interfaces.fsl import ICA_AROMA as aroma
```

Comment:
After messing with this, I realized I need to add ICA_AROMA to the __init__.py in [nipype](https://github.com/nipy/nipype/blob/master/nipype/interfaces/fsl/__init__.py#L7), I'll submit a pull request to fix this.